components/resources/brave_components_strings.grd

```diff
@@ -421,6 +422,8 @@
       <message name="IDS_BRAVE_UI_RECURRING" desc="">Recurring</message>
       <message name="IDS_BRAVE_UI_RECURRING_DONATION" desc="">Monthly Tips</message>
       <message name="IDS_BRAVE_UI_RECURRING_DONATIONS" desc="">Monthly Tips</message>
+      <message name="IDS_BRAVE_UI_REDDIT_TIP_TITLE" desc="">Tip u/{{ user }} for their post:</message>
```

Comment:
we should move `u/` into user variable, so that if this changes we can change it in JS and we don't need to request new translations

Reply:
Not sure if I'm understanding the ask. `u/` is to reddit is `@` is to twitter and that's how it's done above.

Reply:
yeah but that shouldn't be in translation as `u/username` is `user` variable. What can easily happen is that u will be translated

Reply:
moved